packages/office-ui-fabric-react/src/components/DatePicker/examples/DatePicker.Basic.Example.tsx

```diff
@@ -24,6 +24,25 @@ export interface IDatePickerBasicExampleState {
   firstDayOfWeek?: DayOfWeek;
 }
 
+interface IDatePickerExampleObject {
+  datePickerExample: string;
+}
+
+const classNames: IDatePickerExampleObject = mergeStyleSets({
```

Comment:
This isn't relevant here due to the next comment, but in general, you don't need to define an interface for one-off calls like this to `mergeStyleSets`. (If you really wanted to have an interface, it should be called something like `IDatePickerExampleClassNames`.)